src/views/tutorial-view/utils/mdx-components.module.css

```diff
@@ -0,0 +1,3 @@
+.tabsWrapper {
+  margin: 24px 0 48px 0;
```

Comment:
Based on the screenshot, it _looks_ like the spacing below the code block is consistent above each heading, not necessarily something specific to code block. so maybe this isn't accurate? [Margin collapsing](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Mastering_margin_collapsing) might come into play here.

Reply:
Ah yeah great point! 💖 Will switch to `24px` on both top and bottom. I think this is definitely something we'll want to further refine in 🎟️ [Convert global-ish docs content CSS to MDX custom components](https://app.asana.com/0/1201987349274776/1201653123138729/f). I might try to start on that this week since I think it could help us better reason about spacing in MDX content 👍